src/resmom/catch_child.c

```diff
@@ -1159,13 +1164,17 @@ update_jobs_status(void)
 
 	for (pjob = (job *)GET_NEXT(svr_alljobs);
 		pjob; pjob = (job *)GET_NEXT(pjob->ji_alljobs)) {
+			time_now = time(NULL);
+			if (pjob->ji_last_resc_updated > time_now - min_check_poll)	
```

Comment:
Please add parenthesis and use greater/equal as above:
```suggestion
			if (pjob->ji_last_resc_updated >= (time_now - min_check_poll))	
```

Reply:
Done